src/components/Sponsors.astro

```diff
@@ -76,7 +76,7 @@ const logoList: Array<{
   </h2>
 
   <!-- Grid container -->
-  <div class="grid grid-cols-6 gap-5 md:grid-cols-12">
```

Comment:
Why are we removing mobile vs desktop responsiveness? This prevents us from reflowing logos to be denser on desktop and less dense on mobile.

Reply:
We aren't - the responsiveness is taken care of in the individual colspan of each item rather than the overall number of columns in the grid

Reply:
ahh ok that works